experiment/simulation/css/main.css

```diff
@@ -1 +1,363 @@
-/* You CSS goes in here */
\ No newline at end of file
+.window { z-index:20; }
+.jtk-connector { z-index:4; }
+.jtk-endpoint { z-index:5; }
+.jtk-overlay { z-index:6; }
+
+.columns {

```

Comment:
Aren't Bulma columns by default flex?

Reply:
Oh yes, sorry, I used that while debugging. Removed it now.